AmazonKinesisVideoDemoApp/README.md

```diff
@@ -1,54 +1,14 @@
 # Running AmazonKinesisVideoStreaming Sample
 
-More information: [https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/producer-sdk-android.html](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/producer-sdk-android.html)
+## 1. Provision and setup
 
-## 1. Create a user pool
-  * Go to https://console.aws.amazon.com/cognito/
-  * Click `Manage your User Pools`
-  * Click `Create a user pool`
-  * Fill-in `Pool name`
-  * Click `Review defaults`
-  * Click `Create user pool`
-  * Copy `Pool Id` :clipboard:
-  * Select `App clients` in the left nav.
-  * Click `Add an app client`
-  * Fill-in `App client name`
-  * Click `Create app client`
-  * Click `Show details` and copy `App client id` and `App client secret` :clipboard:
-    * ![Shows show details button](screenshots/click_show_details.png) `-->` ![](screenshots/copy_app_client_id_and_secret.png)
+Follow the prerequisites section to provision AWS resources required to run this sample, and assign the appropriate permissions the IAM role: [https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/producer-sdk-android.html](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/producer-sdk-android.html)
 
-## 2. Create an identity pool
-  * Go to https://console.aws.amazon.com/cognito/
-  * Click `Manage Federated Identities`
-  * Click `Create new identity pool`
-  * Fill-in `Identity pool name`
-    * ![Shows field for inputting identity pool name](screenshots/pool_name.png)
-  * Under the heading `Authentication providers`, in the `Cognito` tab, fill-in the `User Pool Id` and `App client id` from the user pools step.
-    ![Shows field for inputting identity pool name](screenshots/fill_in_user_pool.png)
-  * Click `Create create`
-  * There will be details for 2 roles. Look at the one for `authenticated identities` and click `Edit` next to the policy document and your policy should look like this:
-    ```
-    {
-        "Version": "2012-10-17",
-        "Statement": [
-          {
-            "Effect": "Allow",
-            "Action": [
-              "cognito-identity:*",
-              "kinesisvideo:*"
-            ],
-            "Resource": [
-              "*"
-            ]
-          }
-        ]
-      }
-    ```
-  * Click `Allow`
-  * Copy the `Identity Pool Id` from the code snippets on the screen. :clipboard:
+* Amazon Cognito user pool
+* Amazon Cognito identity pool
 
-## 3. Paste
-  * You will need all the information from the above steps that have :clipboard: and paste them into this file on your local copy [awsconfiguration.json](src/main/res/raw/awsconfiguration.json). Here's what it should look like when you're done:
+## 2. Paste
+  * You will need all the information from the step above that you have :clipboard: and paste them into this file on your local copy of [awsconfiguration.json](src/main/res/raw/awsconfiguration.json). Here's what it should look like when you're done:
```

Comment:
looks like all the references to :clipboard: above were deleted in this PR, can you update the instructions here?

Reply:
Done